### ISSUE TITLE
no focus in Downloads window after refresh

### ISSUE THREAD

**Issue by cseppan** [USEPA-OAQPS/emf#58]
Same issue as #57

**Comment by cseppan** [comment]
Same fix as noted in issue #57

Commit 8e748d26c57b275584c7d95b6e97da7aa370724b adds refreshButton.requestFocusInWindow() at the end of the refresh process.

**Comment by cseppan** [comment]
Focus change shouldn't happen on automatic refreshes, only when the user clicks the Refresh button

**Comment by cseppan** [comment]
Commit fb2b477029a9fbb570d8478c20b61f04d7f62d60 updates the code to only request focus on the Refresh button when the Donwloads window is currently selected.
